js/about/aboutActions.js

```diff
@@ -88,6 +88,38 @@ const aboutActions = {
   },
 
   /**
+   * Generates a file with the users backup keys
+   */
+  generateKeyFile: function (backupAction) {
```

Comment:
it would probably be helpful to give these actions/constants names that include `ledger`. Eventually we'll probably break them out into ledger actions